src/Console/Command/FixCommand.php

```diff
@@ -320,7 +320,11 @@ protected function execute(InputInterface $input, OutputInterface $output)
             ->getReporter($resolver->getFormat())
         ;
 
-        $stdErr = ($output instanceof ConsoleOutputInterface) ? $output->getErrorOutput() : 'txt' === $reporter->getFormat() ? $output : null;
+        $stdErr = $output instanceof ConsoleOutputInterface
+            ? $output->getErrorOutput()
+            : ('txt' === $reporter->getFormat() ? $output : null)
+        ;
```

Comment:
@SpacePossum 

```php
($output instanceof ConsoleOutputInterface) ? $output->getErrorOutput() : 'txt' === $reporter->getFormat() ? $output : null;
```
means
```php
(($output instanceof ConsoleOutputInterface) ? $output->getErrorOutput() : 'txt' === $reporter->getFormat()) ? $output : null;
```
not
```php
($output instanceof ConsoleOutputInterface) ? $output->getErrorOutput() : ('txt' === $reporter->getFormat() ? $output : null);
```

-------------------

before my change this always returns `$output`